test/integration/targets/setup_paramiko/aliases

```diff
@@ -1 +1,2 @@
 needs/target/setup_remote_tmp_dir
+skip/freebsd/13.0
```

Comment:
This needs to be done on the test that fails (connection_paramiko_ssh), not the included target (setup_paramiko).

Reply:
done